build/configs/rtl8730e/loadable_apps/defconfig

```diff
@@ -193,11 +197,12 @@ CONFIG_ARCH_HAVE_MMU=y
 # CONFIG_ARCH_NAND_HWECC is not set
 # CONFIG_ARCH_HAVE_EXTCLK is not set
 # CONFIG_ARCH_HAVE_POWEROFF is not set
-# CONFIG_ARCH_HAVE_RESET is not set
+CONFIG_ARCH_HAVE_RESET=y
 CONFIG_ARCH_USE_MMU=y
 CONFIG_ARCH_HAVE_FETCHADD=y
 # CONFIG_ARCH_ADDRENV is not set
 # CONFIG_PAGING is not set
+CONFIG_ARCH_IRQPRIO=y
```

Comment:
same as previous

Reply:
Enabled in all defconfigs, added default selection for Cortex A32 as well.